adding_species_manuscript.tex

```diff
@@ -411,10 +412,11 @@ \section*{Additions to \stdpopsim}
 Among other features, this upgrade includes a discrete-site model of mutation,
 which enables simulating sites with multiple mutations and possibly more than two alleles.
 Another key feature added to \stdpopsim's simulation framework was the ability to model non-crossover recombination.
-In bacteria and archaea, recombination occurs through horizontal transfer of homologous DNA segments from one organism to another \citep{Thomas2005,Didelot2010,Gophna2022}.
-As a result, such species cannot be realistically simulated with a recombination model that considers only crossovers,
+%REVISION: clarified HGT, due to rev 3 comment
+In bacteria and archaea, recombination occurs through horizontal gene transfer, which can add new genetic material from other species (e.g., via the transfer of plasmids) or replace homologous sequences in the chromosome through homologous recombination \citep{Thomas2005,Didelot2010,Gophna2022}.
+Since this mechanism is fundamentally different from crossover recombination, such species cannot be realistically simulated with a recombination model that considers only crossovers,
```

Comment:
Maybe this is a gap in my understanding, but since crossover recombination can be between homologous chromosomes (and I've seen that described as "homologous recombination," I'm not sure homologous recombination and crossover recombination are "fundamentally different." Maybe "fundamentally different from meiotic recombination"?

Reply:
I think we can remove the "fundamentally different comment, because I agree with you that it's not entirely different or separate. Maybe just say:
> As a result, such species ...
(without weighing in on the difference between the mechanisms)